receiver.go

```diff
@@ -72,6 +72,9 @@ func (r *Receiver) IssueCredit(credit uint32) error {
 
 // DrainCredit sets the drain flag on the next flow frame and
 // waits for the drain to be acknowledged.
+//
+// If the context is cancelled or its deadline expires, the
```

Comment:
I think this should just be considered a link error.

Reply:
Reverted per discussion.

Reply:
DrainCredit has been removed.